src/index.html

```diff
@@ -13,6 +13,39 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The class names `calendar--start-day-sun` and `calendar--month-length-31` suggest that you're trying to control the start day and the length of the month through the HTML. This should be controlled in your JavaScript logic instead. You can then apply the appropriate classes to the calendar element dynamically based on the current month.